src/components/Compare/Compare.style.tsx

```diff
@@ -359,21 +354,40 @@ export const DataCellValue = styled.span<{
   text-transform: capitalize;
 `;
 
+export const VaccinationsCell = styled.div`
+  display: flex;
+  flex-direction: row;
+  min-width: 110px;
+`;
+
+export const VaccinationsCellValue = styled.div`
+  font-size: 16px;
+  font-family: Source Code Pro;
+  color: ${COLOR_MAP.GRAY_BODY_COPY};
+  margin-right: 0.5rem;
+  text-transform: capitalize;
+`;
+
+export const VaccinationsCellProgressBar = styled.div`
+  display: flex;
+  width: 100%;
+  margin: auto 0.25rem;
+  padding-right: 20px;
+`;
+
 export const DataCell = styled.td<{
-  $iconColor: string;
   $isSelected: boolean;
+  $desiredWidthPercent: number;
+  $minWidthPx: number;
 }>`
   ${SharedCellStyles}
-  min-width: ${metricCellWidth}px;
+  width: ${props => props.$desiredWidthPercent}%;
+  min-width: ${props => props.$minWidthPx}px;
   padding-right: 0;
   color: ${({ $isSelected }) =>
     $isSelected ? 'black' : `${COLOR_MAP.GRAY_BODY_COPY}`};
   font-weight: ${({ $isSelected }) => $isSelected && '600'};
   background-color: ${({ $isSelected }) => $isSelected && 'rgba(0,0,0,0.02)'};
-
-  svg {
-    color: ${({ $iconColor }) => $iconColor};
-  }
```

Comment:
This is now specified on the `svg` element itself since I need to specify the color within the column's render() method which is used as the contents of the `<DataCell>` container...